bin/connect-to-db

```diff
@@ -1,3 +1,3 @@
 #!/bin/bash 
 
- mysql -h speechql.cutq0x5qwogl.us-east-1.rds.amazonaws.com -P 3306 -u gabe -p
+ mysql -h speechql.cutq0x5qwogl.us-east-1.rds.amazonaws.com -P 3306 -u gabe -p --local-infile=1
```

Comment:
the local infile thing lets you insert cvs on the client machine to the server (by default its not allowed)